MWSE/ScriptUtilLua.cpp

```diff
@@ -51,9 +51,9 @@ namespace mwse::lua {
 		lua_mwscript["addItem"] = [](sol::optional<sol::table> params) {
```

Comment:
This function is deprecated. We don't recommend using the lua `mwscript.*` API. No significant changes should be made to it.

Reply:
Sure, just thought it might have been used previously when it still allowed for LEVIs, but looking at the lua dump doesn't look like it. This could probably be reverted then.